slackbridge/bots.py

```diff
@@ -150,24 +163,27 @@ def signedOn(self):
             self.join('#{}'.format(channel['name']))
 
     def privmsg(self, user, channel, message):
-        # user is like 'jvperrin!Jason@fireball.ocf.berkeley.edu' so only
+        self.post_to_slack(user, channel, message)
+
+    def action(self, user, channel, message):
+        self.post_to_slack(user, channel, '_{}_'.format(message))
+
+    def post_to_slack(self, user, channel, message):
+        # User is like 'jvperrin!Jason@fireball.ocf.berkeley.edu' so only
         # take the part before the exclamation mark for the Slack display name
         assert user.count('!') == 1
-        user_nick, _ = user.split('!')
+        user, _ = user.split('!')
```

Comment:
Minor, but I feel that this should be a method in utils.

Reply:
Yeah, that would make sense and also make this easier to actually run tests on. I think if we can move more stuff into utils or other helper methods then that will be good overall.

Reply:
Alright, I split this off into a separate utils function.